lib/auth/tun.go

```diff
@@ -992,7 +992,7 @@ func (c *TunClient) getClient() (client *ssh.Client, err error) {
 		if trace.IsAccessDenied(err) {
 			return nil, trace.Wrap(err)
 		}
-		log.Debugf("%v.getClient() throttle auth server %v: %v", c, authServer, err)
+		log.Errorf("%v.getClient() error while connecting to auth server %v: %v: throttling", c, authServer, err)
```

Comment:
maybe we should remove "throttling" part? even I don't understand what it means...